tests/console/zypper_lr.pm

```diff
@@ -15,8 +15,7 @@ sub run() {
     my $self = shift;
 
     become_root;
-    script_run("zypper lr -d; echo zypper-lr-\$? > /dev/$serialdev");
-    wait_serial("zypper-lr-0") || die "zypper lr failed";
+    assert_script_run("zypper lr -d");
```

Comment:
I recommend to do less changes at once. It looks like you can replace many "script_run + wait_serial" by "assert_script_run" but this should be separated from getting rid of wait_idle